deploy/0.Mocks.js

```diff
@@ -10,6 +10,12 @@ module.exports = async ({ getNamedAccounts, deployments, getChainId }) => {
             contract: 'MockERC20',
             args: ['yAxis', 'YAX', 18]
         });
+        await deploy('COMP', {
+            from: deployer,
+            log: true,
+            contract: 'MockERC20',
+            args: ['Compound', 'COMP', 18]
+        });
```

Comment:
Can we deploy this in `6.5.StrategyIdle.js`? I think we should only deploy shared mocks in `0.Mocks.js`.